packages/smart-contracts/packages/nft/scripts/data/levelsData.json

```diff
@@ -80,8 +80,8 @@
         "supply_cap": -1,
         "base_uri": "https://ipfs.io/ipfs/QmPoii7N1Ddn8dRg8915wMnd7VdedvS4PkivzU5KAJRRpm/",
         "standard_projects_quantity": -1,
-        "premium_projects_quantity": 0,
-        "limited_projects_quantity": 0
```

Comment:
"premium_projects_quantity": 0,
"limited_projects_quantity": 0
all zero values should stay zero values.
Maybe slight miscommunication happened